source/DSCResources/Resources/Vsphere.VmHostVssPortGroupSecurity.ps1

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+$rules = $stig.RuleList | Select-Rule -Type 'VspherePortGroupSecurityRule'
+
+foreach ($vm in $vmGroup)
+{
+    foreach ($rule in $rules)
+    {
+        if ($rule.AllowPromiscuous)
+        {
+            $allowPromiscuousInherited = $rule.AllowPromiscuousInherited
```

Comment:
Instead of having all these if statements can we put the $rule.*Inherited in the resource block below and not assign it to a variable first?

Reply:
the rule comes from all the VspherePortGroupSecurityRule rules in this case.  It's looping through several rules to assign those variables.  The $rule isn't available outside the foreach where these if statements are located.